packages/web/services/__tests__/technology.js

```diff
@@ -508,11 +508,11 @@ describe('getTechnologies', () => {
 		});
 	});
 
-	test('it returns false if request fails', async () => {
+	test('it returns empty if request fails', async () => {
```

Comment:
```suggestion
	test('it returns an empty array if request fails', async () => {
```

Acredito que assim o teste fica mais fácil de ler ao bater o olho, deixando explícito o que é`empty`